modules/yarn.lock

```diff
@@ -1411,115 +1288,6 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@emotion/cache@^10.0.27":
```

Comment:
Huh, similar changes to those in https://github.com/liferay-frontend/liferay-portal/pull/660... I wonder how all these modules crept in and why are they leaving now... 🤔

Reply:
I wonder if it has to do with using the `install-scripts.sh` from the frontend-projects repo. I did that earlier today and wonder if it created any stale items in the yarn cache.

Reply:
deduped yarn.lock and looks a little better now

Reply:
nvm, still looks wild